test-case/check-signal-stop-start.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+##
+## Case Name: check-signal-stop-start
+## Preconditions:
+##    N/A
+## Description:
+##    Run aplay/arecord on each pipeline and use SIGSTOP and SIGCONT to
+##    simulate keyboard inputs ctrl+z and fg.
+##    kill -SIGSTOP $pid will stop the thread
+##    kill -SIGCONT $pid will start the thread
+## Case step:
+##    1. aplay/arecord on PCM
+##    2. send SIGSTOP/SIGCONT to stop/start
+## Expect result:
+##    no errors for aplay/arecord
+##
+
+source $(dirname ${BASH_SOURCE[0]})/../case-lib/lib.sh
+
+OPT_OPT_lst['t']='tplg'     OPT_DESC_lst['t']='tplg file, default value is env TPLG: $TPLG'
+OPT_PARM_lst['t']=1         OPT_VALUE_lst['t']="$TPLG"
+
+OPT_OPT_lst['i']='sleep'    OPT_DESC_lst['i']='interval time for stop/start'
+OPT_PARM_lst['i']=1         OPT_VALUE_lst['i']=0.5
+
+OPT_OPT_lst['d']='duration' OPT_DESC_lst['d']='duration time'
+OPT_PARM_lst['d']=1         OPT_VALUE_lst['d']=10
+
+OPT_OPT_lst['s']='sof-logger'   OPT_DESC_lst['s']="Open sof-logger trace the data will store at $LOG_ROOT"
+OPT_PARM_lst['s']=0             OPT_VALUE_lst['s']=1
+
+func_opt_parse_option $*
+
+tplg=${OPT_VALUE_lst['t']}
+count=${OPT_VALUE_lst['c']}
```

Comment:
'c' is not defined. There is no use of it, either.  Please enable the support of $count number of start/stop.

In the meantime, do you need to specify duration in this test? can you let it run indefinitely and kill aplay/arecord once you reached the $count?